datahub-kubernetes/datahub/charts/datahub-ingestion-cron/values.yaml

```diff
@@ -9,34 +9,40 @@ image:
 
 imagePullSecrets: []
 
-crons: []
+crons: {}
   #### Example data
-  ## Metadata ingestion name
-  ##
-  #name: "crawler"
-
-  ## Daily at midnight (we may want to offset this to not conflict with other processes)
-  #schedule: "0 0 * * *"
-
-  ## Deployment pod host aliases
-  ## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
-  ##
-  #hostAliases: []
-
-  ## Environment variables.
-  #env: {}
-
-  ## Environment variables from Secret resources.
-  #envFromSecrets: {}
-
-  ## Additional primary volume mounts
-  ##
-  #extraVolumeMounts: []
-
-  ## Additional primary volumes
-  ##
-  #extraVolumes: []
-
-  ## Add your own init container or uncomment and modify the given example.
-  ##
-  #extraInitContainers: {}
\ No newline at end of file
+  #hive:
+    ## Daily at midnight (we may want to offset this to not conflict with other processes)
+    #schedule: "0 0 * * *"
+
+    ## Command to be executed
+    #command: ["/bin/sh", "-c", "datahub ingest -c config.yml"]
+
+    ## Deployment pod host aliases
+    ## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
+    ##
+    #hostAliases: []
+
+    ## Environment variables.
+    #env: {}
+
+    ## Environment variables from Secret resources.
+    #envFromSecrets: {}
+
+    ## Additional primary volume mounts
+    ##
+    #extraVolumeMounts:
```

Comment:
Since recipes are central to running datahub ingest. Can we make that explicit instead of having to add extraVolumes and extraVolumeMounts every time?

Reply:
I agree with the idea, not sure how to implement it though. Any suggestions?

Reply:
I think a recipe field in this map would do. It can have the volume name, mount path, and we can explicitly create a volume mount and volume using this information. 

In case more configurations are needed, they can be added as extraVolumes and extraVolumeMounts

WDYT?

Reply:
Do you mean have the recipe yaml content itself for each cron definition in the values.yaml of this chart?
If yes, have some reservations.
1.) it would make this values.yaml significantly larger as you add more and more cron jobs.
2.) it may confuse users who require multiple recipes for one cron job type. This was my original use-case.

Reply:
Oh no. Just the configmap name and filename inside the cnofigmap. 

Want to make sure folks using this chart knows that they need to add a configmap with the recipe.

Reply:
Got it! I’ll update the PR tomorrow as suggested.

Reply:
@pedro93 : gentle ping on this one :)

Reply:
My apologies for taking so long, work has been chaotic. 
Please take a look and see if it matches what your expectations are. I'm not confident that the proposed UX for the non-trivial path is the best, when a simple recipe does not suffice but could not come up with a better solution.